content/en/real_user_monitoring/guide/browser-sdk-upgrade.md

```diff
@@ -10,9 +10,193 @@ further_reading:
   text: "Use Datadog Session Replay to view real-time user journeys"
 ---
 
-## Overview 
+## Overview
 
-Follow this guide to migrate between major versions of the RUM and Logs Browser SDKs.
+Follow this guide to migrate between major versions of the Browser RUM and Browser Logs SDKs.
+
```

Comment:
Let's add something to point people to the latest SDK version, something like:

For the latest browser SDK capabilities, go [here](https://docs.datadoghq.com/real_user_monitoring/browser/).